app/src/main/res/layout/profile_progress_header.xml

```diff
@@ -50,6 +50,7 @@
       android:text="@{viewModel.profile.name}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="20sp"
+      android:textAlignment="viewStart"
```

Comment:
Ditto

Reply:
ok

Reply:
<img width="427" alt="test" src="https://user-images.githubusercontent.com/55504132/103403018-bfbc2480-4b74-11eb-8725-f2feb751e251.png">

Can you pl. clarify that is it totally  correct  ?

Reply:
@lakshaygarg128 Yes, it looks good.